---a/net/minecraft/command/EntityNotFoundException.java
+++b/net/minecraft/command/EntityNotFoundException.java
@@ -1,17 +1,15 @@
 package net.minecraft.command;
 
-public class EntityNotFoundException extends CommandException
-{
+public class EntityNotFoundException extends CommandException {
+
     private static final String __OBFID = "CL_00002335";
 
-    public EntityNotFoundException()
-    {
+    public EntityNotFoundException() {
         this("commands.generic.entity.notFound", new Object[0]);
     }
 
-    public EntityNotFoundException(String p_i46035_1_, Object ... p_i46035_2_)
-    {
+    public EntityNotFoundException(String p_i46035_1_, Object... p_i46035_2_) {
         super(p_i46035_1_, p_i46035_2_);
     }
-}
+}
